src/features/components/atoms/card/card-primary/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Typography from '../../typography'
+import Flex from '../../flex-box'
+import Link from '../../link'
+import Image from '../../image'
+import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
+import dclsx from 'features/utils/dclsx'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+
+type CardItemsProps = {
+    header: TString
+    description: TString
+    icon?: string
+    link?: `/${string}`
+}
+
+const CardPrimary = ({ header, description, icon, link }: CardItemsProps) => {
+    const { is_mobile_or_tablet } = useBreakpoints()
+
+    return (
+        <Flex.Box
+            className={dclsx('item_container')}
+            direction="col"
+            gap="4x"
+            radius="8x"
+            lg={{ padding: '12x' }}
+            padding="8x"
+        >
+            {icon && <Image src={icon} width={48} height={48} pb="4x" lg={{ pb: '8x' }} />}
+
+            <Typography.Heading as="h3" size={is_mobile_or_tablet ? 'xs' : 'xxs'}>
+                <Localize translate_text={header} />
+            </Typography.Heading>
+            <Typography.Paragraph size="small" md={{ mb: '10x' }}>
+                <Localize translate_text={description} />
+            </Typography.Paragraph>
+            {link && (
+                <Flex.Box pt="12x">
+                    <Link
+                        url={{ type: 'internal', to: link }}
+                        size="medium"
+                        textcolor="brand"
+                        font_family="UBUNTU"
+                    >
+                        <Localize translate_text="_t_Learn more >_t_" />
```

Comment:
For the Arrow ">" we are currently using <ImageWithDireciton src={Arrow} alt="arrow right" />
on the current implementation because the arrow is not the same size as text. You could check that and if not needed could go with your approach

Reply:
i think we are using ">" in homepage under live price and trade types. It's fine i guess.

Reply:
Okay 👍